conan/tools/gnu/autotoolsgen.py

```diff
@@ -1,37 +0,0 @@
-import platform
```

Comment:
I removed this but not because I don't like the aggregator, but the way it was creating a single `conanbuildenv.bat` made it difficultly to "split" the generators and keep the build helpers working. See the other comment in the build helper where I'm not convinced either about the best way to wrap the command with all the available environment scripts.